cambiatus/frontend
# Enhancement/transfers

## What issue does this PR close
Closes #581, closes #579, closes #204 (that's an old one!!)

## Changes Proposed ( a list of new changes introduced by this PR)
- Use the new transfer API from cambiatus/backend#176
- Use new design for the transfers list on the Dashboard
- Introduce a localized date viewer with support to show stuff like "Today", "Yesterday", and any other custom string (if we choose to), along with a localized general date, like "Claimed on 7 june 2021". For this we needed a couple of things:
  - A new `Language` type for better organization
  - A custom element to display the formatted date (so we can use the nice JS api)
  - `Shared` now holds a `Time.Zone` (which solves #204)
- Fix some issues with `Profile.Summary` in a couple of pages
- Add a linter for our CSS. This way we have linters for all languages we use (JS, CSS and Elm), and have a nice standard
- Introduce a new customElement, `infiniteList`, which is used to have things like feeds from social media, where more data is requested automatically as the user scrolls
- Roll back the hotfix for v1.11.1

## How to test ( a list of instructions on how to test this PR)

- Test according to the description #204 
- Make sure the transfer list shows up on the dashboard, and you can access a specific transfer (either by notifications, or by clicking on one on the dashboard)
- See if the dashboard design is according to the figma link in #579 
- Check if you see anything weird with regards to dates (such as on claims, transfers, etc.)
